File: Arrays/Python/Operating_an_array.py
# Your task is to complete all
# three functions

# if the element is found in the list
# function  must return true or else
# return false
def searchEle(a, x):
    # Code here
    if x in a:
        return True
    else:
        return False

# fucntion must return true if 
# insertion is successful or else
# return false
def insertEle(a, y, yi):
    # Code here
    a.insert(yi,y)
    return True

# fucntion must return true if 
# deletion is successful or else
# return false
def deleteEle(a, z):
    # Code here
    if z in a:
        a.remove(z)
    return True
 

File: Solutions/Java/Operating_an_array.java
class GfG {		
	public boolean searchEle(int a[],int x)
       {
		//add code here.
		for(int i=0; i<a.length; i++){
            if (a[i]==x) {
        return true;
            }
		}
    return false;
       }
	public boolean insertEle(int a[],int y,int yi)
        {
            //add code here.
            for (int i=0; i<100; i++) {
                if (i == yi) {
                    a[i] = y;
                    return true;
                }
            }
        return false;
	}
	public boolean deleteEle(int a[],int z)
        {
		//add code here.
		int index=0;
        boolean num=false;
        for(int i=0; i<100; i++) {
            if(a[i]==z) {
                index = i;
                num=true;
            }
        }
        if(num == true) {
            for(int i=99; i>index; i--) {
                a[i-1] = a[i];
            }
        }
        return true;
	}
}
